packages/demos/package.json

```diff
@@ -41,7 +41,8 @@
   "dependencies": {
     "@acronis-platform/icons": "workspace:*",
     "@acronis-platform/ui-component-library": "workspace:*",
-    "npm-run-all": "4.1.5"
+    "npm-run-all": "4.1.5",
+    "vue-router": "4.4.0"
```

Comment:
i am not sure that we should stick to vue-router here

Reply:
but how i can show an example with a router if a router shouldn't exist in demos?

Reply:
yes, we can use _vue-router_ in demos, as it would be some kind of real world usage of ui components